lib/src/scss-shared/_nx-colors.scss

```diff
@@ -72,8 +72,8 @@ $nx-header-border: #f1f3fa;
 
 //  Text
 $nx-heading-color: #2C2C2C;
-$nx-text-color: #474747;
-$nx-text-color-light: $nx-slate;
+$nx-text-color: #2c2c2c;
```

Comment:
Does that usage of #474747 that we switched to `$nx-text-color` in the other PR need to be updated to `$nx-text-color-light` here?

Reply:
> Does that usage of #474747 that we switched to `$nx-text-color` in the other PR need to be updated to `$nx-text-color-light` here?

Yes: https://github.com/sonatype/sonatype-react-shared-components/pull/34/commits/c9d9a4482410d10bfc33ccca51aa2ab3d80066cd